test/e2e/common/projected.go

```diff
@@ -172,13 +172,26 @@ var _ = framework.KubeDescribe("Projected", func() {
 
 	It("optional updates should be reflected in volume [Conformance] [Volume]", func() {
 
-		// We may have to wait or a full sync period to elapse before the
-		// Kubelet projects the update into the volume and the container picks
-		// it up. This timeout is based on the default Kubelet sync period (1
-		// minute) plus additional time for fudge factor.
-		const podLogTimeout = 300 * time.Second
-		trueVal := true
+		// We SecretManager, we may have to wait up to full sync period + TTL of
+		// a secret to elapse before the Kubelet projects the update into the volume
+		// and the container picks it ip.
+		// This timeout is based on default Kubelet sync period (1 minute) plus
+		// maximum secret TTL (based on cluster size) plus additional time for fudge
```

Comment:
Can we get this from the node annotation, instead of hard coding the logic here?